### ISSUE TITLE
Pre-Rendered md-Dialog shown not visible 

### ISSUE THREAD

**Issue by sanujgandhi** [angular/material#9036]
**Actual Behavior**:
 -  `What is the issue? *`
 - 'ARIA: Attribute " aria-label ", required for accessibility, is missing on node: 
<md-dialog role=​"dialog" tabindex=​"-1" id=​"dialogContent_168" aria-describedby=​"dialogContent_168" class=​"md-transition-in" style>​</md-dialog>`

 -  `What is the expected behavior?`
 -  `<div class="md-dialog-container" id="myDialog" style="top: 0px; height: 388px;">
      <div class="_md-dialog-focus-trap" tabindex="0"></div><md-dialog layout-padding="" class="layout-padding _md _md-transition-in" role="dialog" tabindex="-1" id="dialogContent_12" aria-describedby="dialogContent_12" aria-label="Pre-Rendered Dialog ..." style="">
        <h2>Pre-Rendered Dialog</h2>
        <p>
          This is a pre-rendered dialog, which means that <code>$mdDialog</code> doesn't compile its
          template on each opening.
          <br><br>
          The Dialog Element is a static element in the DOM, which is just visually hidden.<br>
          Once the dialog opens, we just fetch the element from the DOM into our dialog and upon close
          we restore the element back into its old DOM position.
        </p>
      </md-dialog><div class="_md-dialog-focus-trap" tabindex="0"></div>
    </div>`

**CodePen** (or steps to reproduce the issue): *
 -  `http://plnkr.co/edit/nCtFiKVJhy7rZfApLE0r?p=preview`
 -  `I think it is unable to find the div? I saw an empty Dialog with error`

**Angular Versions**: *
"angular": "~1.4.9",
"angular-animate": "~1.4.9",
"angular-aria": "~1.4.9",
"angular-messages": "~1.4.9",
"angular-cookies": "~1.4.9",
"angular-material": "~1.0.3",
"angular-ui-router": "~0.2.17",
"material-design-lite": "~1.0.6",

**Additional Information**:
 -  `Browser Type: *`
 -  `Browser Version: *`
 -  `OS: *`
 -  `Stack Traces:`

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by sanujgandhi** [comment]
ARIA: Attribute " aria-label ", required for accessibility, is missing on node: 
<md-dialog role=​"dialog" tabindex=​"-1" id=​"dialogContent_168" aria-describedby=​"dialogContent_168" class=​"md-transition-in" style>​</md-dialog>​

**Comment by sanujgandhi** [comment]
http://plnkr.co/edit/nCtFiKVJhy7rZfApLE0r?p=preview

**Comment by sanujgandhi** [comment]
"angular": "~1.4.9",
    "angular-animate": "~1.4.9",
    "angular-aria": "~1.4.9",
    "angular-messages": "~1.4.9",
    "angular-cookies": "~1.4.9",
    "angular-material": "~1.0.3",
    "angular-ui-router": "~0.2.17",
    "material-design-lite": "~1.0.6",

**Comment by sanujgandhi** [comment]
I am using this in ms-form-wizard

**Comment by DevVersion** [comment]
@sanujgandhi Please fulfill the issue template, otherwise we won't be able to solve your issue.

**Comment by sanujgandhi** [comment]
@DevVersion Hello Sir, I am new to Angular Material, I don't know what I am doing wrong. I have updated some of fields as are in my Knowledge. I am using <md-tab> and in one tab page I want to use Pre-rendered md-dialog but I get the upper error. 
Thanks

**Comment by DevVersion** [comment]
@sanujgandhi Well, that's no problem. 

The problem with your demo is, that you're using a _super_ old version of ngMaterial, which doesn't support Pre-Rendererd Dialogs.

Here is an updated demo with `master`
> http://plnkr.co/edit/m3qMzWTgVQuxs5aRq2bp?p=preview

**Comment by mmumshad** [comment]
@DevVersion The plunker doesn't work.  I have the same issue with pre-rendered dialogue. I am using Material 1.0.8. 

@sanujgandhi were you able to solve this problem?

**Comment by ThomasBurleson** [comment]
@mmumshad - we only support last release (currently v1.1.1) and HEAD.

**Comment by mmumshad** [comment]
Thank you!! Latest version works!

**Comment by DevVersion** [comment]
@mmumshad The plunker didn't work because I deleted this Plunker a long time ago.
